components/brave_wallet_ui/common/async/base-query-cache.ts

```diff
@@ -381,9 +320,7 @@ export class BaseQueryCache {
       return token.logo
     }
 
-    return token.logo.startsWith('ipfs://')
-      ? (await this.getIpfsGatewayTranslatedNftUrl(token.logo)) || ''
-      : `chrome://erc-token-images/${token.logo}`
+    return `chrome://erc-token-images/${token.logo}`
```

Comment:
can we move this function out from the base cache now that it no longer needs an async call into core? Maybe to asset-utils?

Reply:
I thought about that change in context of my current task: https://github.com/brave/brave-browser/issues/37735

I think we should  not remove an ability to display NFT which have a address like: `ipfs://ipfs://Qm...7MA` (I know it works for now, because we have IPFS schema support and tranlsation IPFS address on redirect)

I mean we should not change that lines, because when we remove local IPFS node and IPFS schema at all, we will not translate IPFS addresses automatically like it works now. We will need to do it manually (by call `this.getIpfsGatewayTranslatedNftUrl(token.logo)`).
For example here is my draft PR: https://github.com/brave/brave-core/blob/35025563bf90a24b76e4ddca3b974435b9023f5b/components/brave_wallet_ui/common/async/base-query-cache.ts#L373-L375, I leaved that place unchanged.

I mean in case of NFT's address like `ipfs://ipfs://Qm...7MA`  we should call this.getIpfsGatewayTranslatedNftUrl(token.logo) for translation from  `ipfs://ipfs://Qm...7MA` to  `https://ipfs.io/ipfs/Qm...7MA`

WDYT?

Reply:
@supermassive  I agree, let's keep this function as is. I will make the change